backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,146 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor._
+import cromwell.backend.model.{JobDescriptor, WorkflowDescriptor}
+
+import scala.concurrent.Future
+import scala.util.{Success, Failure}
+
+object BackendActor {
+
+  // Commands
+  sealed trait BackendActorMessage
+  case class Execute(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case class Abort(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case object AbortAll extends BackendActorMessage
+  case class Recover(jobDescriptor: JobDescriptor) extends BackendActorMessage
+
+  // Events
+  sealed trait BackendActorEvent
```

Comment:
Forgive me if I didn't understand this properly, but on the call yesterday it sounded like form a contract perspective the event messages are irrelevant.  They are not part of the contract.  If so, lets move them out of of here since the are important only to specific backends.  If they're here they're part of the contract.